packages/tradershub/src/hooks/useSyncLocalStorageClientAccounts.ts

```diff
@@ -0,0 +1,60 @@
+import { useCallback } from 'react';
+import { useLocalStorage, useReadLocalStorage } from 'usehooks-ts';
+import { getAccountsFromLocalStorage } from '@/utils';
+import { useActiveTradingAccount, useCreateNewRealAccount, useSettings } from '@deriv/api';
+
+type TNewTradingAccount = NonNullable<ReturnType<typeof useCreateNewRealAccount>['data']>;
+
+/**
+ * @name useSyncLocalStorageClientAccounts
+ * @description A custom hook that syncs the client accounts to the local storage
+ * @returns { addTradingAccountToLocalStorage: (newAccount: TNewTradingAccount) => void }
+ */
+const useSyncLocalStorageClientAccounts = () => {
+    const { data } = useActiveTradingAccount();
+    const { data: settingsData } = useSettings();
+    const [, setLocalStorageClientAccounts] = useLocalStorage(
+        'client.accounts',
+        useReadLocalStorage('client.accounts') ?? {}
+    );
+
+    const addTradingAccountToLocalStorage = useCallback(
+        (newAccount: TNewTradingAccount) => {
+            if (newAccount && data) {
+                const dataToStore = {
+                    accepted_bch: 0,
+                    account_category: 'trading',
+                    account_type: 'standard',
+                    balance: 0,
```

Comment:
@shahzaib-deriv why we are hardcoding this?

Reply:
Will update. Thanks @mahdiyeh-deriv for the suggestion

Reply:
@shahzaib-deriv it's not resolved yet

Reply:
@mahdiyeh-deriv balance is being set as 0 however the `account_category` & `account_type` has been updated to be used from the API response.